pyproject.toml

```diff
@@ -68,6 +68,7 @@ protobuf  = ">=3.12.0"
 pyarrow = "7.0.0"
 pyjwt  = "^2.0.0"
 rich = ">=11"
+requests = ">=2.28.0"
```

Comment:
If `requests` are not used in favour of `aiohttp`,  this line should be removed.

Reply:
Can we reduce the lower bound? It is effectively latest or nothing at the moment and this might clash with anything else that has a lower upper bound. Colab has `2.23.0` for instance.

Reply:
In the future, we should stick to only one http client library.